backend/SC4SNMP_UI_backend/common/conversions.py

```diff
@@ -47,6 +47,27 @@ def ui2backend(self, document: dict, **kwargs):
 class ProfileConversion(Conversion):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.__backend2ui_conditional_operations = {
+            "lt": "less than",
+            "gt": "greater than",
+            "equal": "equal",
+            "in": "in"
+        }
+        self.__ui2backend_conditional_operations = {}
+        for key, value in self.__backend2ui_conditional_operations.items():
+            self.__ui2backend_conditional_operations[value] = key
+
+        self.__backend2ui_profile_types = {
+            "field": "smart"
+        }
+        self.__ui2backend_profile_types = {}
+        for key, value in self.__backend2ui_profile_types.items():
+            self.__ui2backend_profile_types[value] = key
+
+    def __string_value_to_numeric(self, value: str):
+        if value.isnumeric():
+            value = int(value)
```

Comment:
it's better to `try` `except` here instead, `isnumeric` checks if the values is numerical, so if you have float like `0.01` it will return `false`

Reply:
Maybe something like this would be more clear?
```    
try:
  if value.isnumeric():
    return int(value)
  else:
    return float(value)
  except ValueError:
    return value
```
I'll leave it to your personal judgement, pick the version you like more :)

Reply:
Changed